client/src/components/Onboarding/OnboardingAdmin.module.scss

```diff
@@ -122,30 +147,27 @@ tbody {
 }
 
 .action-link {
+  padding: 0;
+
   font-size: 12px;
 }
+.action-link:not(:first-child) { margin-left: 0.25em; }
+.action-link:not(:last-child) { margin-right: 0.25em; }
 
 .staffwait {
   background-color: var(--global-color-accent--light);
 }
 
-.root {
-  th:nth-child(1),
-  td:nth-child(1) {
-    width: 15%;
-  }
-  th:nth-child(2),
-  td:nth-child(2) {
-    width: 20%;
-  }
-  th:nth-child(3),
-  td:nth-child(3) {
-    width: 15%;
-  }
-  th:nth-child(4),
-  td:nth-child(4) {
-    width: 20%;
-  }
+.users {
```

Comment:
**Explanation**.

Changed from `.root` to `.users` because:
1. These styles are specific to the users `<table>`, not the root `<div>`.
2. Adding the `table-layout` made sense to do here with the column widths.
3. The diff, here specifically, is easier to view as a single big +/– (than many smaller +/–'s)